.scripts/automation_generate.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+track2-codegen-automation-for-pipeline --inputJsonPath=$1 --outputJsonPath=$2 --use=@autorest/typescript@6.0.0-beta.5
```

Comment:
@dw511214992 I realize this is a very old PR but after looking through it I have a few questions and suggestions.

- Where do we publish track2-codegen-automation-for-pipeline? It should probably be scripts in this repo
- Can we please move these scripts under eng/scripts?
- Can we please move the swagger_to_sdk_config.json under eng directory?

Reply:
Hi @weshaggard 

- Answer for question 1:  the codes of `track2-codegen-automation-for-pipeline` is in [DevOps Repository](https://devdiv.visualstudio.com/DevDiv/_git/azure-track2-js-sdk-release-tools) now, and I will move it to js sdk repository in next month (I don't have time to do it this month, and I'll do it once I have time. Sorry about it)
- Answer for question 2: I agree to put `automation_generate.sh` in eng/scripts, and I'll do it. However, the tool `track2-codegen-automation-for-pipeline` not only contains script for swagger pipeline, but also contains other tools, such as generating changelog, bumping version automatically, generate codes automatically in local. So I think it's better to put it in `common/tools`. 
- Answer for question 3: I don't think we can move `swagger_to_sdk_config.json` to other places because the swagger pipeline asks this file should be in the root of js sdk repository.
Thanks

Reply:
Thanks @dw511214992 

1) Cool yes I would like to get that tool moved into the repo and if possible under the eng folder. I'm trying to get more of the common/tools folder under eng as well.
2) It is perfectly fine for all the script automation to be in the JS repo we have lots of similar scripts already. Part of the reason we want it in the repo is so we can share stuff where possible.
3) You can move it to another location but it requires a change in in the specs repo. See https://github.com/Azure/azure-rest-api-specs/blob/main/specificationRepositoryConfiguration.json#L35 as an example. Ideally we would also not call the config "track2" either, I know a number of others already are so I won't worry much but I generally try to avoid code names like track 2 and instead use the versioning of the tool instead.

Reply:
@weshaggard thanks, and I'll do these tasks together. About moving the scripts to eng folder, the code in [DevOps Repository](https://devdiv.visualstudio.com/DevDiv/_git/azure-track2-js-sdk-release-tools) is complex (it's a project) and I'm not sure whether moving it to eng folder is a good solution. Anywhere, I'll rethink about it.